wordpress-mobile/WordPress-Android
# View all Referrers screen

This PR adds a new View all Referrers screen.

To test:
1. Go to Stats
2. Switch to a DWMY tab
3. Tap the View more button of the Referrers card
4. Notice a list of all referrers is loaded
